packages/web/src/validators/validator.ts

```diff
@@ -104,6 +98,20 @@ export function requiredValidator(element, type, subtype?) {
           element.validator = ['requiredRadio'];
         }
         break;
+      case 'checkboxGroup':
+        if (element.validator) {
+          element.validator.unshift('requiredCheckboxGroup');
+        } else {
+          element.validator = ['requiredCheckboxGroup'];
+        }
+        break;
+      case 'singleCheckbox':
```

Comment:
Very small naming suggestion to keep the checkboxes grouped together a bit more through their name.

```suggestion
      case 'checkboxSingle':
```